src/components/CustomLayout.tsx

```diff
@@ -1,11 +1,12 @@
-import React, { FC, ReactNode } from "react";
+import React, { DependencyList, FC, ReactNode } from "react";
 import classnames from "classnames";
 
 interface Props {
   header?: ReactNode;
   children: ReactNode;
   mainClassName?: string;
   contentClassName?: string;
+  dependencies?: DependencyList;
```

Comment:
We don't need the dependencies here, do we?

Reply:
no we don't, this should have been removed, will remove